components/tidb_query/src/expr/builtin_encryption.rs

```diff
@@ -137,6 +138,21 @@ impl ScalarFunc {
         }
         Ok(Some(i64::from(LittleEndian::read_u32(&input[0..4]))))
     }
+
+    #[inline]
+    pub fn random_bytes<'a, 'b: 'a>(
+        &'b self,
+        ctx: &mut EvalContext,
+        row: &[Datum],
+    ) -> Result<Option<Cow<'a, [u8]>>> {
+        let length = try_opt!(self.children[0].eval_int(ctx, row));
+        if length < 1 || length > MAX_RAND_BYTES_LENGTH {
+            ctx.warnings
```

Comment:
It’s better to attach the length information to warning message to make error message more clearly.

Reply:
It seems that there is no parameter detail information in other functions of tikv. And I checked the tidb Sig it also not have detail information.

Reply:
And it seem incorrect_parameters fn design only for print the fn name.
```rust
    pub fn incorrect_parameters(val: &str) -> Error {
        let msg = format!(
            "Incorrect parameters in the call to native function '{}'",
            val
        );
        Error::Eval(msg, ERR_INCORRECT_PARAMETERS)
    }
```

Reply:
How about:
```rust
let msg = format!("Incorrect parameters {} in the call to native function 'random_bytes'", length);
ctx.warnings.append(Error::Eval(msg, ERR_INCORRECT_PARAMETERS))
```

Reply:
How about refactor the incorrect_parameters fn?

Reply:
@breeswish  Can I refactor the incorrect_parameters fn?

Reply:
Yeah, feel free to do it :)